dds/idl/be_util.cpp

```diff
@@ -98,6 +98,7 @@ void
 be_util::usage (void)
 {
   ACE_DEBUG((LM_DEBUG,
+    ACE_TEXT(" --default-nested treat unannotated nodes as if they were nested\n")
```

Comment:
`ACE_TEXT(" --default-nested\ttreat unannotated types as if they were nested\n")`

- node is an internal term, doesn't make much sense in user documentation.
- a tab to line up the description with the one after it